src/main/java/com/notably/logic/parser/NewCommandParser.java

```diff
@@ -61,6 +66,7 @@ public NewCommandParser(Model notablyModel) {
 
         AbsolutePath path = ParserUtil.createAbsolutePath(title, notablyModel.getCurrentlyOpenPath());
         commands.add(new OpenCommand(path));
+        logger.info("OpenCommand created");
```

Comment:
Perhaps you can be more explicit here. There are a lot of paths of execution in this `NewCommandParser#parse` execution. It'll be better if you can fine-tune the logger to log out the exact detail given the different paths that are taken!